.github/workflows/docker_test.yml

```diff
@@ -26,39 +62,36 @@ jobs:
     runs-on: ubuntu-latest
     name: Test docker images
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out the code at a specific ref
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.nightly_tag_main || github.ref }}
```

Comment:
this should be replaced with the `tag`:
```
ref: ${{ inputs.tag || github.ref }}
```

(Can even just leave as `ref: ${{ inputs.tag }}` actually, since we've required that it exist.

Reply:
Have I mentioned how complicated having the separate `base` package is?

So, what actually needs to happen here (and everywhere else) is that since we've bundled `langflow-base` with `langflow-main`, we _always_ need a ref from `langflow-main` in order to check out the code for `langflow-base`. We don't have separate tags or a package for `langflow-base`. So what that means, when we're trying to build `langflow-base` for a specific tag, we _must_ have already released a `langflow-main` at a specific tag that points to the `langflow-base` we want to build. 

So, my earlier comment about having a single `tag` is not correct. We need to have something like "tag_to_check_out" and "tag_we_want_to_build_for". 

This whole process of the main and base split needs some investigation.

Reply:
And to illustrate the issue: if I passed in `tag:v1.0.15`, we're currently going to check out whatever the `github.ref` is, which defaults to wherever the action was run from (so, probably `main:latest`). So we'd be building `v1.0.x` into an incorrectly tagged docker image.